Institutional Certification Metadata

patrick-lloyd-ray opened the pull request:
This PR contains the initial version of the BICAN Institutional Certification Schema.

patrick-lloyd-ray committed:
initial commit, tables and schema

patrick-lloyd-ray committed:
updating README

lydiang commented:
Can we rename this project from Informed Consent -> Institutional Certification?
This will help limit the scope of this model to only focus on communication of the information in a Institutional Certification form rather than a later domain of Informed Consent?

lydiang committed:
Update IC_value_set.csv

Adding some definition from https://sharing.nih.gov/genomic-data-sharing-policy/institutional-certifications/completing-an-institutional-certification-form#step-0

lydiang committed:
Update IC_value_set.csv

lydiang committed:
Update IC_value_set.csv

lydiang committed:
Update IC_Metadata_Schema.csv

Update with description from IC NIH site
https://sharing.nih.gov/genomic-data-sharing-policy/institutional-certifications/completing-an-institutional-certification-form#step-0

patrick-lloyd-ray committed:
updating README

patrick-lloyd-ray committed:
changing title

patrick-lloyd-ray committed:
Updating Schema and Value set

patrick-lloyd-ray committed:
Updating Institutional_Certification_README.md

patrick-lloyd-ray commented:
Approvals

- [x] @patrick-lloyd-ray 
- [x] @lydiang 
- [x] @rightbower 
- [x] @nsuvarnaiari

patrick-lloyd-ray commented:
@nsuvarnaiari to add GSR field from NIMP

patrick-lloyd-ray committed:
reorganizing the content to improve clarity and structure

patrick-lloyd-ray committed:
Update README.md

patrick-lloyd-ray committed:
updating main README.md to reflect recent changes in metadata schemas and documentation.

patrick-lloyd-ray committed:
resolving conflicts in the following files:
- README.md
